src/Discord.Net.WebSocket/Entities/Channels/SocketDMChannel.cs

```diff
@@ -108,6 +129,13 @@ internal SocketMessage RemoveMessage(ulong id)
             => _messages?.Remove(id);
 
         //Users
+        /// <summary>
+        ///     Gets a user in this channel from the passed <paramref name="id"/>.
+        /// </summary>
+        /// <param name="id">The id of the user you want to get.</param>
+        /// <returns>
+        ///     The recipient, or the CurrentUSer otherwise; <c>null</c>
```

Comment:
* Be more descriptive. e.g. `An user object that represents the recipient of this channel`
* Inconsistent casing
* The entire return statement punctuation is confusing.